.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 dist: precise
 jdk:
-- openjdk8
```

Comment:
openjdk 8 definitely exists

Reply:
I think you have a typo in your yaml

Reply:
Travis error indicates it got the version number and it is outside range:
>Expected feature release number in range of 9 to 14, but got: 8

Is there another method to grab it?

Reply:
Can you try dropping `dist`? That might be the cause

Reply:
that did it